src/markdown-pages/add-ons/rook.md

```diff
@@ -36,11 +30,17 @@ flags-table
 
 ## Block Storage
 
-For production clusters, Rook should be configured to use block devices rather than the filesystem.
-Enabling block storage is required with version 1.4.3+. Therefore, the `isBlockStorageEnabled` option will always be set to true when using version 1.4.3+.
-The following spec enables block storage for the Rook add-on and automatically uses disks matching the regex `/sd[b-z]/`.
-Rook will start an OSD for each discovered disk, which could result in multiple OSDs running on a single node.
-Rook will ignore block devices that already have a filesystem on them.
+For Rook versions 1.4.3 and later, block storage is required.
+For Rook versions earlier than 1.4.3, block storage is recommended in production clusters.
+
+You can enable and disable block storage for Rook with the `isBlockStorageEnabled` field in the kURL spec.
+By default, the `isBlockStorageEnabled` field is set to `true` for Rook versions 1.4.3 and later.
+
+When the `isBlockStorageEnabled` field is set to `true`, Rook starts an OSD for each discovered disk.
```

Comment:
```suggestion
When the `isBlockStorageEnabled` field is set to `true`, or when using Rook versions 1.4.3 and later, Rook starts an OSD for each discovered disk.
```

Reply:
Same question as above: It sounds like it's _not_ that the `isBlockStorageEnabled` field is set to `true` by default in 1.4.3 and later, but rather that the field no longer exists in 1.4.3 and later. Is that right?